internal/util/importutil/import_wrapper.go

```diff
@@ -0,0 +1,393 @@
+package importutil
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"os"
+	"path"
+	"strconv"
+
+	"github.com/milvus-io/milvus/internal/allocator"
+	"github.com/milvus-io/milvus/internal/common"
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/proto/schemapb"
+	"github.com/milvus-io/milvus/internal/storage"
+	"github.com/milvus-io/milvus/internal/util/typeutil"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	JSONFileExt  = ".json"
+	NumpyFileExt = ".npy"
+)
+
+func printFieldsDataInfo(fieldsData map[string]storage.FieldData, msg string) {
+	stats := make([]zapcore.Field, 0)
```

Comment:
we need more info here, such as which segmentid?what collection? or which path we are gonna to flush to